revenue_app/urls.py

```diff
@@ -1,7 +1,7 @@
 from django.conf.urls import url
-from .views import Index
+from .views import Dashboard
```

Comment:
Unless the path is _too_ long, it's usually a good idea to use absolute paths as opposed to relative paths.

Reply:
see #8